build/Targets/Packages.props

```diff
@@ -230,7 +230,7 @@
     <SystemXmlXmlDocumentVersion>4.3.0</SystemXmlXmlDocumentVersion>
     <SystemXmlXmlSerializerVersion>4.3.0</SystemXmlXmlSerializerVersion>
     <SystemXmlXPathXDocumentVersion>4.3.0</SystemXmlXPathXDocumentVersion>
-    <SQLitePCLRawbundle_greenVersion>1.1.11</SQLitePCLRawbundle_greenVersion>
+    <SQLitePCLRawbundle_greenVersion>1.1.2</SQLitePCLRawbundle_greenVersion>
```

Comment:
we only need to revert this. not whole thing...?

Reply:
The other changes require this change

Reply:
are we reverting whole thing to be safe?

Reply:
(As I understand it)

Reply:
The `ReadBlob(byte[])` APIs were added in 1.1.10, so it would not build otherwise.